iothub/device/src/Transport/Mqtt/MqttIotHubAdapter.cs

```diff
@@ -297,26 +297,20 @@ private async Task ConnectAsync(IChannelHandlerContext context)
                     Debug.Assert(_mqttTransportSettings.ClientCertificate != null);
                 }
 
-                // This check is added to enable the device or module client to available plug and play features. For devices or modules that pass in the model Id,
-                // the SDK will enable plug and play features by using the PnP-enabled service API version, and appending the model Id to the MQTT CONNECT packet (in the username).
-                // For devices or modules that do not have the model Id set, the SDK will use the GA service API version.
-                string serviceParams = null;
-                if (string.IsNullOrWhiteSpace(_options?.ModelId))
-                {
-                    serviceParams = ClientApiVersionHelper.ApiVersionQueryStringLatest;
-                }
-                else
-                {
-                    serviceParams = $"{ClientApiVersionHelper.ApiVersionQueryStringPreview}&{ModelIdParam}={_options.ModelId}";
-                }
-
-                string usernameString = $"{this._iotHubHostName}/{id}/?{serviceParams}&{DeviceClientTypeParam}={Uri.EscapeDataString(this._productInfo.ToString())}";
+                string usernameString = $"{this._iotHubHostName}/{id}/?{ClientApiVersionHelper.ApiVersionQueryStringLatest}&{DeviceClientTypeParam}={Uri.EscapeDataString(this._productInfo.ToString())}";
```

Comment:
`this.`?

Reply:
yeah, can be removed for sure. I'll update this.